crates/agent/src/thread_store.rs

```diff
@@ -731,7 +731,7 @@ pub enum SerializedMessageSegment {
         signature: Option<String>,
     },
     RedactedThinking {
-        data: Vec<u8>,
```

Comment:
It should be fine to change this, since in practice we were never storing redacted thinking in the first place.